### PR TITLE
Tests/initialize unit tests

### PR DESCRIPTION
**Pull request by ekelen**
It's rebased 👍

### CODE REVIEW

File: internal/player/player_test.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package player
+
+import "testing"
+
+func TestCreatePlayer(t *testing.T) {
+	tables := []struct {
```
Comment (eskombro) [inline internal/player/player_test.go:6]: The idea would be to use the player struct. You can use it directly here as we are in the same package. If not, then when we modify the player struct, we need to come and change this one too.
tables := Player{
...
}

**Review by eskombro — COMMENTED** [review]
1 comment to check ;)

File: internal/player/player_test.go (hunk 1 above)
Comment (ekelen) [reply internal/player/player_test.go:6]: Agreed, will do

Comment (ekelen) [reply internal/player/player_test.go:6]: Done, I think. Unrelated thing for later: If we have a public constructor for Player (and create some public accessors), maybe make Player type private?

**Review by eskombro — COMMENTED** [review]
Alles Gut! :)

**Review by eskombro — APPROVED** [review]

**Pull request merged**